src/components/table/DatasetSnapshotsTable.jsx

```diff
@@ -39,20 +39,23 @@ class DatasetSnapshotsTable extends React.PureComponent {
     const columns = [
       {
         label: 'Snapshot Name',
-        property: 'name',
+        name: 'name',
+        allowSort: true,
         render: (row) => (
-          <Link to={`/snapshots/details/${row.id}`} className={classes.jadeLink}>
+          <Link to={`/snapshots/${row.id}/overview`} className={classes.jadeLink}>
```

Comment:
There's no more `overview` route. The snapshot overview pages now live at `/snapshot/{ID}`

Reply:
Oh thanks, I thought I had caught all of the route changes.